Postcard - Data views updates (#586)
osisoft/OCS-Docs: content/api-reference/data-views/get-data-view-data/quick-start-get-data-view-data.md

The supported modes of retrieval are interpolated and stored values within a user-specified range. Several formats are available, such as JSON and CSV.

### Index
All data in a data view is associated with an index value, for example, a timestamp. If there are multiple groups in the data view, the index and grouping field values together form a unique identifier for each data record.

#### Index type
Though `DateTime` is the most common index type for stream data, data views can target data of other index types, as noted in [Define a Data View](xref:DataViewsQuickStartDefine).

> For `DateTime` indexed data views, index values should use ISO 8601 format and can be in either UTC or local time with an offset specified. Data retrieved using the [Data API](xref:DataViewsDataAPI) will be returned in UTC. Interval timespan values should be formatted using the standard timespan format, ie., d.hh:mm:ss.fffffff.

Reviewer: Rewrite this in present tense. Make it active voice as well. Suggestion:

```suggestion
For `DateTime` indexed data views, use the ISO 8601 format  as the index value. Format interval timespan values using the standard timespan format. For example, d.hh:mm:ss.fffffff. Use either UTC or local time with an offset specified. Data retrieved using the [Data API](xref:DataViewsDataAPI) returns in UTC.
```
